### PR TITLE
doc:  Added zustand-interval-persist to third party

### PR DESCRIPTION
**Pull request by YashChauhan858**
Created a third party package that is built on top of zustand and provides interval based saving to store

### CODE REVIEW

File: docs/integrations/third-party-libraries.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ This can be done using third-party libraries created by the community.
 
 - [@colorfy-software/zfy](https://colorfy-software.gitbook.io/zfy/) — 🧸 Useful helpers for state management in React with Zustand.
 - [@davstack/store](https://www.npmjs.com/package/@davstack/store) — A zustand store factory that auto generates selectors with get/set/use methods, supports inferred types, and makes global / local state management easy.
+- [zustand-interval-persist](https://www.npmjs.com/package/zustand-interval-persist) — An enhancement for zustand that enables automatic saving of the store's state to the specified storage at regular interval.
```
Comment (dai-shi) [inline docs/integrations/third-party-libraries.md:19]: Let's order them alphabetically.

File: docs/integrations/third-party-libraries.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ This can be done using third-party libraries created by the community.
...
```
Comment (dai-shi) [inline docs/integrations/third-party-libraries.md:20]: Let's remove the empty line please.

**Review by dai-shi — CHANGES_REQUESTED** [review]
can you make changes?

File: docs/integrations/third-party-libraries.md (hunk 2 above)
Comment (YashChauhan858) [reply docs/integrations/third-party-libraries.md]: fixed with latest commit

File: docs/integrations/third-party-libraries.md (hunk 1 above)
Comment (YashChauhan858) [reply docs/integrations/third-party-libraries.md]: fixed with latest commit

File: docs/integrations/third-party-libraries.md [hunk 3]
```diff
@@ -37,6 +35,7 @@ This can be done using third-party libraries created by the community.
 - [vue-zustand](https://github.com/wobsoriano/vue-zustand) — State management solution for Vue based on Zustand.
 - [zoov](https://github.com/InfiniteXyy/zoov) — State management solution based on Zustand with Module-like API.
 - [zundo](https://github.com/charkour/zundo) — 🍜 Undo and redo middleware for Zustand, enabling time-travel in your apps.
+- [zustand-interval-persist](https://www.npmjs.com/package/zustand-interval-persist) — An enhancement for zustand that enables automatic saving of the store's state to the specified storage at regular interval.
```
Comment (dai-shi) [inline docs/integrations/third-party-libraries.md:38]: Oh, it looks like zundo and zukeeper should be in the other order.
zustand-interval-persist should be between zustand-forms and zustand-middleware-computed-state.

Comment (YashChauhan858) [reply docs/integrations/third-party-libraries.md]: rearranged order for zukeeper and zundo , and placed zustand-interval-persist at correct place.

**Review by dai-shi — APPROVED** [review]
LGTM
Thanks for your contribution!

**Pull request merged**